code/bio_alias.h

```diff
@@ -0,0 +1,24 @@
+//
+// Created by Fábio Domingues on 20/07/17.
+//
+
+#ifndef CRASHMONKEY_BIO_DEFINES_H
+#define CRASHMONKEY_BIO_DEFINES_H
+
+#include <linux/version.h>
+
+#if LINUX_VERSION_CODE < KERNEL_VERSION(3, 16, 0)
+
+#define BI_SIZE                 bi_size
+#define BI_SECTOR               bi_sector
+#define BIO_ENDIO(bio, err)     bio_endio(bio, err)
+
+#else
```

Comment:
nit: please write a comment stating that this was originally written to target 4.4 kernels since there is no kernel version for this segment.